src/main/java/Duke.java

```diff
@@ -1,10 +1,134 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        WelcomeMessage();
+        String line;
+        Scanner in = new Scanner(System.in);
+
+        Task[] toDo = new Task[100];
+        int trackIndex = 0;
+        boolean isProgress = true;
+
+        while (isProgress) {
+            line = in.nextLine();
+
+            if (line.equals("bye")) {
+                isProgress = false;
+                goodbyeMessage();
+            } else if (line.equals("list")) {
+                handleListRequest(toDo, trackIndex);
+            } else if (line.contains("done")) {
+                handleDoneRequest(line, toDo);
+            } else if (line.contains("todo")) {
+                handleToDoRequest(line, toDo, trackIndex);
+                trackIndex++;
+            } else if (line.contains("deadline")) {
+                handleDeadlineRequest(line, toDo, trackIndex);
+                trackIndex++;
+            } else if (line.contains("event")) {
+                handleEventRequest(line, toDo, trackIndex);
+                trackIndex++;
+            } else {
+                handleTaskRequest(line, toDo, trackIndex);
+                trackIndex++;
+            }
```

Comment:
I like how neat this entire if-else block is and how it follows coding standards and is an example of good code quality.